content/blog/kallisti5/2021-02-23_icculus_microgrant_2020.md

```diff
@@ -0,0 +1,11 @@
++++
+type = "blog"
+title = "Icculus Microgrant 2020"
+author = "kallisti5"
+date = "2021-02-23 14:08:15-06:00"
+tags = ["haiku", "software"]
++++
+
+Haiku, Inc. is proud to announce being the target of the [Icculus Microgrant program](https://icculus.org/microgrant/2020/) for 2020. We would like to thank Icculus, and all of the wonderful folks who have donated to Haiku, Inc. over the years.
+
+Your continued donations help us keep us working on what we love, and help us continuously grow and develop the ideal personal computing operating system.
```

Comment:
"Your continued donations help us keep us" should maybe be "Your continued donations help keep us".